library/src/main/java/com/heinrichreimersoftware/materialintro/slide/SimpleSlide.java

```diff
@@ -519,10 +539,48 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                 descriptionView.setTextColor(textColorSecondary);
             }
 
+            if (getActivity() instanceof SimpleSlideActivity) {
+                ((SimpleSlideActivity)getActivity()).onSlideViewCreated(this, fragment, id);
+            }
+
             return fragment;
         }
 
         @Override
+        public void onDestroyView() {
+            if (getActivity() instanceof SimpleSlideActivity) {
+                int id = getArguments().getInt(ARGUMENT_ID);
+                ((SimpleSlideActivity)getActivity()).onSlideDestroyView(this, getView(), id);
+            }
+            innerLinearLayout = null;
+            titleView = null;
+            descriptionView = null;
+            imageView = null;
+            super.onDestroyView();
+        }
+
+        public LinearLayout getInnerLinearLayout() {
+            return innerLinearLayout;
```

Comment:
A bit hacky though 😄 
Basically `SimpleSlide` is the out of the box solution for say 80% of all intros and when you need forms or inputs or custom layouts, you could either use `FragmentSlide` with a `Fragment` or even create a fully customized `Slide`